_sass/icons.scss

```diff
@@ -0,0 +1,56 @@
+@font-face {
+  font-family: 'fontello';
+  src: url('/assets/fonts/fontello.eot?46807042');
+  src: url('/assets/fonts/fontello.eot?46807042#iefix') format('embedded-opentype'),
+       url('/assets/fonts/fontello.woff?46807042') format('woff'),
+       url('/assets/fonts/fontello.ttf?46807042') format('truetype'),
+       url('/assets/fonts/fontello.svg?46807042#fontello') format('svg');
+  font-weight: normal;
+  font-style: normal;
+}
+ 
+ [class^="icon-"]:before, [class*=" icon-"]:before {
+  font-family: "fontello";
```

Comment:
Properties should be ordered display, font-family, -moz-osx-font-smoothing, -webkit-font-smoothing, font-style, font-variant, font-weight, line-height, margin-left, margin-right, speak, text-align, text-decoration, text-transform, width